src/MiniCover/HitServices/Hits.cs

```diff
@@ -46,9 +50,47 @@ public static Hits TryReadFromFile(string file)
 
         public static Hits ConvertToHits(string json)
         {
-            var notMergedHits = JsonConvert.DeserializeObject<HitTestMethod[]>(json)
+            var notMergedHits = JsonConvert.DeserializeObject<HitTestMethod[]>(json, new JsonSerializerSettings {
+                    ContractResolver = new CustomContractResolver()
+                })
                 .SelectMany(method => method.HitedInstructions.Select(id => new Hit(id.Key, id.Value, new[] {method}))).ToArray();
             return new Hits(notMergedHits);
         }
     }
+
+    public class CustomContractResolver : DefaultContractResolver {
```

Comment:
Why is this resolver necessary?

Reply:
you right was from test forgot to remove it